src/main/java/com/googlecode/lanterna/graphics/AbstractTextGraphics.java

```diff
@@ -273,16 +273,29 @@ public TextGraphics putString(int column, int row, String string) {
         }
         string = tabBehaviour.replaceTabs(string, column);
         int offset = 0;
+
+
+        EnumSet<SGR> localModifiers = activeModifiers.clone();
```

Comment:
Since `updateLocalModifiers` doesn't change its parameter but instead works on a cloned Set, this first `.clone()` seems unnecessary.